modules/config.lua

```diff
@@ -19,6 +19,8 @@ config.mobileHost = "localhost"
 config.mobilePort = 12345
 --- Define timeout for Heartbeat in msec
 config.heartbeatTimeout = 7000
+--- Flag which defines whether ATF check all validations for particular expectation or just till the first which fails
```

Comment:
@dboltovskyi `check` > `checks` or `will check`

Reply:
@AByzhynar Corrected in 3dd1985